example/BUILD.bazel

```diff
@@ -9,7 +9,7 @@ latex_document(
         "references.bib",
         "//example/example_class:example_class",
     ],
-    cmd_flags = ["--bibtex-cmd=biber"],
+    #cmd_flags = ["--bibtex-cmd=biber"],
```

Comment:
Remove? There is both bibtex and biber, so we don't want to hardcode the choice of binary internally.

But latexrun, runs the bib task with cwd set to the output directory, which puts a burden on the user to point to it correctly. Perhaps it's better to add a default parameter to the rule api? If there is only two possible options?

On the other hand, if the only option we currently had is biber. Then, I think this option is fine. And the line should be removed.

Reply:
done